SPECS/reaper/CVE-2022-37601.patch

```diff
@@ -1,7 +1,33 @@
-diff --git a/node_modules/loader-utils/lib/parseQuery.js b/node_modules/loader-utils/lib/parseQuery.js
+diff --git a/loader-utils/lib/parseQuery.js b/loader-utils/lib/parseQuery.js
 index 12b3efc6..3dd7cb9b 100644
---- a/node_modules/loader-utils/lib/parseQuery.js
-+++ b/node_modules/loader-utils/lib/parseQuery.js
+--- a/loader-utils/lib/parseQuery.js
++++ b/loader-utils/lib/parseQuery.js
```

Comment:
I'm fairly certain that all of these need to be `[ab]/node_modules/...`.

If you look at the spec file, the [patch happens](https://github.com/microsoft/CBL-Mariner/blob/b9b57423006ce180019deaf7e9d021a2f892d7ee/SPECS/reaper/reaper.spec#L111) in this context:
```
cd %{_builddir}/%{srcdir}
echo "Installing src caches"
pushd ./src/ui
echo "Installing bower_components"
tar xf %{SOURCE1}

echo "Installing npm_modules"
tar fx %{SOURCE2}
patch -p1 {PATCH0}
popd
```
So when the patch is applied, we're in `<root-rpm-build-dir>/src/ui`, so the patch file needs paths to reflect that, and have `node_modules` as their base.

I did a quick test and with this file as is (and the `patch` command fixed), `rpmbuild -bc` hangs on the patch command, asking what file to patch. Changing it as I suggest, it patches all three files.